src/resmom_win/start_exec.c

```diff
@@ -2901,60 +2883,17 @@ start_process(task *ptask, char **argv, char **envp, bool nodemux)
 		exec_bail(pjob, JOB_EXEC_FAIL_PASSWORD, NULL);
```

Comment:
https://github.com/PBSPro/pbspro/blob/06f24712f1b101a386d5b88656a24489250e52f0/src/resmom_win/start_exec.c#L2882

should we use log_event here ?